src/test/java/teammates/test/cases/automated/FeedbackSessionRemindEmailWorkerActionTest.java

```diff
@@ -46,14 +46,15 @@ public void allTests() throws Exception {
 
         String[] submissionParams = new String[] {
                 ParamsNames.SUBMISSION_FEEDBACK, session1.getFeedbackSessionName(),
-                ParamsNames.SUBMISSION_COURSE, session1.getCourseId()
+                ParamsNames.SUBMISSION_COURSE, session1.getCourseId(),
+                ParamsNames.USER_ID, "idOfInstructor1OfCourse1"
```

Comment:
See if you can get `idOfInstructor1OfCourse1` from the data bundle instead of hard-coding.